stumpy/__init__.py

```diff
@@ -49,6 +50,10 @@
     from .core import _gpu_aampdist_driver_not_found as gpu_aampdist  # noqa: F401
     from .core import _gpu_stimp_driver_not_found as gpu_stimp  # noqa: F401
     from .core import _gpu_aamp_stimp_driver_not_found as gpu_aamp_stimp  # noqa: F401
+
+    core._gpu_searchsorted_left = core._gpu_searchsorted_left_driver_not_found
```

Comment:
This feels disconnected from the `from . import core` above

Reply:
Yeah...I noticed that! I am waiting to see how the test suite goes. Then, I will test gpu modules on my other PC tomorrow, and update my main and merge it to this branch and take care of these small things. Then, I will push again :)